src/Symfony/Component/Messenger/Command/FailedMessagesRemoveCommand.php

```diff
@@ -78,7 +78,7 @@ private function removeMessages(string $failureTransportName, array $ids, Receiv
             try {
                 $envelope = $receiver->find($id);
             } finally {
-                $this->phpSerializer?->enableClassNotFoundCreation(false);
+                $this->phpSerializer?->disableClassNotFoundCreation();
```

Comment:
what about allowPhpIncompleteClass? that's even more explicit I think

Reply:
acceptPhpIncompleteClass / rejectPhpIncompleteClass ?

Reply:
Sounds good 👍 I’ll update the PR tonight